Sources/Commands/SwiftTestTool.swift

```diff
@@ -138,6 +138,10 @@ struct TestToolOptions: ParsableArguments {
     @Option(help: "Test the specified product.")
     var testProduct: String?
 
+    /// Generate LinuxMain entries and exit.
+    @Flag(name: .customLong("testable-imports"), inversion: .prefixedEnableDisable, help: "Enable or disable testable imports. Enabled by default.")
+    var enableTestableImports: Bool = true
```

Comment:
ideas for a different name welcome. the use facing feature is testable imports so this is what I went with.